WhisperAPI/WhisperAPI.Tests/Unit/ModelValidationTest.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Abstractions;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Routing;
+using Moq;
+using NUnit.Framework;
+using WhisperAPI.Controllers;
+using WhisperAPI.Models.Queries;
+using WhisperAPI.Services.Context;
+using WhisperAPI.Tests.Data.Builders;
+
+namespace WhisperAPI.Tests.Unit
+{
+    [TestFixture]
+    public class ModelValidationTest
+    {
+        private readonly InMemoryContexts _contexts;
+        private readonly ContextController _contexyController;
+
+        public ModelValidationTest()
+        {
+            this._contexts = new InMemoryContexts(new TimeSpan(1, 0, 0, 0));
+            this._contexyController = new ContextController(this._contexts);
+        }
+
+        [Test]
+        public void When_max_documents_is_negative_return_model_state_invalid()
+        {
+            var query = SuggestionQueryBuilder.Build.WithChatKey(new Guid("0f8fad5b-d9cb-469f-a165-708677289502")).WithMaxDocuments(-2).Instance;
```

Comment:
I'll remove all the WithChatKey(new Guid("0f8fad5b-d9cb-469f-a165-708677289502")). I don't need the chatkey to have this value, the builder already do a Guid.newGuid()